apps/pie-docs/src/__tests__/_utilities/__snapshots__/colors.test.js.snap

```diff
@@ -1,39 +1,39 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`colors.js convertHexcodeToRBG converts hexcode "#000" to the expected RGB object 1`] = `
-Object {
+exports[`colors.js > convertHexcodeToRBG > converts hexcode %p to the expected RGB object #000 1`] = `
```

Comment:
This still says `RBG`

Reply:
That's the same as before. The only bit that was incorrect was the function name, not the test name :)

Reply:
But the test name must also be incorrect if it still says RBG 😄

Reply:
Ah I see, if you've undone the other corrections then that's fine